Explorer/Assets/DCL/PluginSystem/World/SceneUIPlugin.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Arch.SystemGroups;
+using DCL.Optimization.Pools;
+using DCL.PluginSystem.World.Dependencies;
+using DCL.SDKComponents.SceneUI.Components;
+using DCL.SDKComponents.SceneUI.Utils;
+using ECS.ComponentsPooling.Systems;
+using ECS.LifeCycle;
+using System.Collections.Generic;
+using UnityEngine.UIElements;
+using UITextInstantiationSystem = DCL.SDKComponents.SceneUI.Systems.UIText.UITextInstantiationSystem;
+using UITextReleaseSystem = DCL.SDKComponents.SceneUI.Systems.UIText.UITextReleaseSystem;
+using UITransformInstantiationSystem = DCL.SDKComponents.SceneUI.Systems.UITransform.UITransformInstantiationSystem;
+using UITransformParentingSystem = DCL.SDKComponents.SceneUI.Systems.UITransform.UITransformParentingSystem;
+using UITransformReleaseSystem = DCL.SDKComponents.SceneUI.Systems.UITransform.UITransformReleaseSystem;
+using UITransformSortingSystem = DCL.SDKComponents.SceneUI.Systems.UITransform.UITransformSortingSystem;
+using UITransformUpdateSystem = DCL.SDKComponents.SceneUI.Systems.UITransform.UITransformUpdateSystem;
+
+namespace DCL.PluginSystem.World
+{
+    public class SceneUIPlugin : IDCLWorldPluginWithoutSettings
+    {
+        private readonly UIDocument canvas;
+        private readonly IComponentPoolsRegistry componentPoolsRegistry;
+
+        public SceneUIPlugin(
+            UIDocument canvas,
+            StyleSheet canvasStyleSheet,
+            ECSWorldSingletonSharedDependencies singletonSharedDependencies)
+        {
+            this.canvas = canvas;
+            if (this.canvas.rootVisualElement != null)
```

Comment:
Yep, I added this null check because some tests that failed, but I've noticed the problem was that I wasn't intantiated correctly the `ScenesUIRootCanvas` asset in the `IntegrationTestsSuite`.

I have just fixed here: https://github.com/decentraland/unity-explorer/pull/204/commits/50c59db4e594780e0a11c7fca633898d9ed160ac